friends/src/App.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import { Route, withRouter } from 'react-router-dom';
+import axios from 'axios';
+
+import FriendList from './components/FriendList';
+import NewFriend from './components/NewFriend';
+import EditFriend from './components/EditFriend';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+        friends: [],
+        currentFriend: []
+    };
+
+    this.addNewFriendHandleSubmit = this.addNewFriendHandleSubmit.bind(this);
+    this.editFriendHandleSubmit = this.editFriendHandleSubmit.bind(this);
+    this.youAreNotMyFriend = this.youAreNotMyFriend.bind(this);
+    this.passProperties = this.passProperties.bind(this);
+  }
+
+  componentDidMount() {
+      const self = this;
+      axios.get('http://localhost:5000/friends')
+          .then(function (response) {
+
+              const friendArr = [];
+              response.data.map(friend => {
+                  return friendArr.push([friend.id, friend.name, friend.age, friend.email])
+              });
+
+              self.setState({
+                  friends: friendArr
+              });
+          })
+          .catch(function (error) {
+              console.log(error);
+          });
+  }
+
+  youAreNotMyFriend(e, id) {
```

Comment:
By converting your class methods to arrow functions you don't have to bind them in the constructor above.
```suggestion
  youAreNotMyFriend = (e, id) => {
```

Reply:
Thanks doe that I remembered I did this before but couldn't quite remember how. I'll go ahead and do the change